pkg/kn/commands/flags/sink.go

```diff
@@ -149,7 +149,7 @@ func parseSink(sink string) (string, string, string) {
 // SinkToString prepares a sink for list output
 func SinkToString(sink duckv1.Destination) string {
 	if sink.Ref != nil {
-		if sink.Ref.Kind == "Service" {
+		if sink.Ref.Kind == "Service" && sink.Ref.APIVersion == "serving.knative.dev/v1" {
```

Comment:
Instead of api version, is it enough to check for the group, like sink.Ref.Group  = "serving.knative.dev" ?

Reply:
I'm afraid not. For example, a trigger with sink is a ksvc will have sink.Ref.Group = ""

Reply:
looks like Group is not stabilized, more details here -> https://github.com/knative/eventing/issues/5086. But if you include the version in the check, it would need to be changed when a new knative serving version comes.

Reply:
looks like Group is not stabilized, more details here -> knative/eventing#5086. But if you include the version in the check, it would need to be changed when a new knative serving version comes.